pkg/morph/client/notary.go

```diff
@@ -423,19 +420,25 @@ func (c *Client) NotarySignAndInvokeTX(mainTx *transaction.Transaction) error {
 		multiaddrAccount.PrivateKey().SignHashable(uint32(magicNumber), mainTx)...,
```

Comment:
Please add an issue to refactor this in future, transactions should be signed by notary actor. It may be not yet ready for this, but this will change.